paddle/majel/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(MAJEL_CXX_FILES place.cpp)
```

Comment:
> NOTE: The suffix of unit test files should strictly limit to `.cpp`, because it doesn't contain any CUDA syntaxes and semantics.

I agree that if a source file doesn't include CUDA code, the suffix shouldn't be .cu.  But I personally prefer `.cc` over `.cpp` following Google C++ Style Guide.